app/migrations/gorm-migrate.go

```diff
@@ -4,29 +4,38 @@ import (
 	"encoding/json"
 	"log"
 	"os"
-	"star-pos/features/user"
-	userData "star-pos/features/user/repository"
+	"star-pos/app/databases"
+	userModel "star-pos/features/user/model"
+	"star-pos/features/user/repository"
 
 	"gorm.io/gorm"
 )
 
-func InitMigration(db *gorm.DB) {
+func InitMigration() {
 	file, err := os.ReadFile("userdummy.json")
 	if err != nil {
 		log.Fatal("error reading file: ", err)
 	}
-	var users []user.UserCore
+	var users []userModel.User
 	err = json.Unmarshal(file, &users)
 	if err != nil {
 		log.Fatal("error unmarshalling JSON: ", err)
 	}
 
-	repositoryInterface := userData.New(db)
-
 	for _, data := range users {
-		err = repositoryInterface.Insert(data)
-		if err != nil {
-			log.Fatal("error insert data: ", err)
+		var existingUser userModel.User
+		result := databases.DB.Where("phone_number = ?", data.PhoneNumber).First(&existingUser)
```

Comment:
we just need to check whether the table is empty or not.
please move this checking above the loop, and run this query `select count(id) from users`. if the result > 0 then we skip dummy data insertion